compiler/src/iree/compiler/Codegen/Utils/GPUUtils.h

```diff
@@ -206,6 +206,13 @@ IREE::GPU::TargetAttr getGPUTargetAttr(Operation *op);
 /// Returns std::nullopt if none found.
 std::optional<int> getGPUSubgroupSize(mlir::FunctionOpInterface func);
 
+/// Returns a map of supported MMA intrinsic instructions based on the
+/// GPU target descriptions in `moduleOp`. Each entry in the map associates
+/// an `Operation*` ( an `IREE::HAL::ExecutableVariantOp`) with a
+/// vector of `IREE::GPU::MMAIntrinsic` attributes.
+llvm::SmallDenseMap<Operation *, SmallVector<IREE::GPU::MMAIntrinsic>>
```

Comment:
Why do we want to return `Operation *` instead of `ExecutableVariantOp`? I think this would simplify both code and the documentation.